CMakeLists.txt

```diff
@@ -82,24 +84,13 @@ message(STATUS "TBB system found?  ${THRUST_TBB_FOUND}")
 message(STATUS "OMP system found?  ${THRUST_OMP_FOUND}")
 
 if (THRUST_CUDA_FOUND)
-  include(cmake/ThrustCUDAConfig.cmake)
+  include(cmake/ThrustCudaConfig.cmake)
 endif()
 
-if ("MSVC" STREQUAL "${CMAKE_CXX_COMPILER_ID}")
-  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 19.00)
-    message(FATAL_ERROR "This version of MSVC no longer supported.")
-  endif ()
-endif ()
-
-if ("GNU" STREQUAL "${CMAKE_CXX_COMPILER_ID}")
-  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.4)
-    message(FATAL_ERROR "This version of GCC no longer supported.")
-  endif ()
-endif ()
-
```

Comment:
This moved somewhere else?

Reply:
It's out of date and we check for this at compile time now.